frontend/common/services/useAccount.ts

```diff
@@ -0,0 +1,72 @@
+import { Res } from 'common/types/responses'
+import { Req } from 'common/types/requests'
+import { service } from 'common/service'
+
+export const accountService = service
+  .enhanceEndpoints({ addTagTypes: ['Account'] })
+  .injectEndpoints({
+    endpoints: (builder) => ({
+      deleteAccount: builder.mutation<Res['account'], Req['deleteAccount']>({
+        invalidatesTags: [{ id: 'LIST', type: 'Account' }],
+        query: (query: Req['deleteAccount']) => ({
+          body: {
+            current_password: query.current_password,
+            delete_orphan_organisations: query.delete_orphan_organisations,
+          },
+          method: 'DELETE',
+          url: `auth/users/me`,
```

Comment:
Ah, this is an annoying one - we need to be specific about the trailing slashes here. 

```suggestion
          url: `auth/users/me/`,
```